docs/quick-start/index.md

```diff
@@ -682,19 +685,240 @@ public Server(String serverName) {
+`spine.enableJava().server()` in your Gradle project.</p>
+
+Then, the `Client` class declares a field for keeping subscriptions to the results of a command
+execution. We'll see how this field is used in a minute. 
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="private @Nullable ImmutableSet" end="private @Nullable ImmutableSet"?>
+```java
+private @Nullable ImmutableSet<Subscription> subscriptions;
+```
+
+First of all, let's see how the `Client` instances are created.
+
+### The constructor
+
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="public Client(" 
+             end="    }"?>
+```java
+public Client(String serverName) {
+    this.client = inProcess(serverName)
+            .shutdownTimout(2, TimeUnit.SECONDS)
+            .build();
+}
+```
+
+Again, similarly to a `Server`, a `Client` is created using in-process connection to a named server.
+The `shutdownTimeout` parameter configures the amount of time allowed for completing current
+client-server communications before closing the client. 
+
+Now, let's review the main thing the `Client` class does, sending the `Print` command.
+
+### Sending the command
+<?embed-code file="examples/hello/src/main/java/io/spine/helloworld/client/Client.java" 
+             start="sendCommand() {" 
+             end="    }"?>
+```java
+public void sendCommand() {
+    String userName = System.getProperty("user.name");
+    Print commandMessage = Print.newBuilder()
+            .setUsername(userName)
+            .setText("Hello World!")
+            .vBuild();
+    this.subscriptions =
+            client.asGuest()
+                  .command(commandMessage)
+                  .observe(Printed.class, this::onPrinted)
+                  .post();
+}
+```
+
+The method does two things:
+   1. Creates a command message using the login name of the current computer user.
+   2. Posts the command, subscribing to the `Printed` event which will be generated as the result
+      of the handling the posted command.
+      
+Let's review the second thing in details.
+   
+The `client.asGuest()` call starts composing a client request. For the sake of the simplicity,
+we on do this on behalf of a user who is not logged in. A real-world application would send
+start sending most of the commands using the `client.onBehalfOf(UserId)` call.
+
+The `command(commandMessage)` call tells we want to send a command with the passed message.
+
+Then, we subscribe to the `Printed` event which would be generated as the result of handling
+the command. The events obtained from the server would be passed to the `onPrinted()` method
+of the `Client` class.
+
+The `post()` method sends the command to the server, returning the set with one `Subscription`
+to the `Printed` event. We store the returned set in the field, so that it can be used later 
```

Comment:
Let's simplify:
"We store the returned set in the field to use later for canceling the subscription."

Note I have omitted the cancellation part here since you speak of it just a few sentences down below.